recipes/aws-kvs-pic/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,14 @@
+#include <stdio.h>
+#include "com/amazonaws/kinesis/video/client/Include.h"
```

Comment:
I may be doing something wrong here or this lib may be broken. 
I'm including a header from the `client` component and I'm calling a random function declared in that header. I'd expect that linking to the `client` module to be enough, but it fails with a lot of undefined symbols.
Looking at the sources it looks like the `kvspic` library is a superset of the other 3. I asked on their repo and they confirmed that, but the little libraries should work as standalone libs IIUC.
https://github.com/awslabs/amazon-kinesis-video-streams-pic/issues/128

Reply:
Also in the sources there are a lot of lines that link this libraries to nothing. So they might be broken?
E.g. https://github.com/awslabs/amazon-kinesis-video-streams-pic/blob/master/CMakeLists.txt#L142